dask_cudf/core.py

```diff
@@ -1357,4 +914,50 @@ def reduction(args, chunk=None, aggregate=None, combine=None,
         if isinstance(arg, _Frame):
             dsk.update(arg.dask)
 
-    return new_dd_object(dsk, b, meta, (None, None))
+    return dd.core.new_dd_object(dsk, b, meta, (None, None))
+
+
+@dd.core.get_parallel_type.register(cudf.DataFrame)
+def get_parallel_type_dataframe(_):
+    return DataFrame
+
+
+@dd.core.get_parallel_type.register(cudf.Series)
+def get_parallel_type_series(_):
+    return Series
+
+
+@dd.core.get_parallel_type.register(cudf.Index)
+def get_parallel_type_index(_):
+    return Index
+
+
+@dd.core.meta_nonempty.register(cudf.Series)
+def meta_nonempty_series(x):
+    s = dd.core.meta_nonempty(x.to_pandas())
+    # TODO: return cudf.Series.from_pandas(s)
```

Comment:
Is there any limitations on the cuDF side preventing you from doing this today?

Reply:
This was a holdover from the last time we did this, before that method existed.  Replaced.  (thanks for implementing it then!)